documentation/bolt_installing_modules.md

```diff
@@ -214,7 +215,9 @@ To specify a git module, use the following keys in the specification:
 | Key | Description | Required |
 | --- | --- | :-: |
 | `git` | The URI to the GitHub repository. URI must begin with either `https://github.com`or `git@github.com`. | ✓ |
+| `name` | The name of the module. This name is used to name the module in the Puppetfile and the directory that the module's contents are downloaded to. Required if `resolve` is `false`. | |
```

Comment:
This should be more explicit about whether it's namespaced or not.

Reply:
Updated. Not 100% sure on the phrasing, though... for example, if the name doesn't match the module's _actual_ name, tasks can still be run but not plans.